File: python/activity_diagram.py
def _hire_developers() -> None:
    raise NotImplementedError("Not implemented yet")


def _create_game() -> None:
    raise NotImplementedError("Not implemented yet")


def _assign_developers(*developers: str) -> None:
    raise NotImplementedError("Not implemented yet")


def _release_game() -> None:
    raise NotImplementedError("Not implemented yet")


def _review_game() -> None:
    raise NotImplementedError("Not implemented yet")


def _deserves_award() -> bool:
    raise NotImplementedError("Not implemented yet")


def _create_award() -> None:
    raise NotImplementedError("Not implemented yet")


def main() -> None:
    # Start
    _hire_developers()
    _create_game()
    _assign_developers("Henrik", "Olle", "Jozef")
    _release_game()
    _review_game()
    # Award?
    if _deserves_award():
        # Yes
        _create_award()
    # No

    # End


if __name__ == "__main__":
    main()

File: java/ActivityDiagram.java
public class ActivityDiagram {
    private static void hireDevelopers() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static void createGame() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static void assignDevelopers(String... developers) {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static void releaseGame() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static void reviewGame() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static boolean deservesAward() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    private static void createAward() {
        throw new UnsupportedOperationException("Not implemented yet");
    }
    
    public static void main(String[] args) {
        // Start
        hireDevelopers();
        createGame();
        assignDevelopers("Henrik", "Olle", "Jozef");
        releaseGame();
        reviewGame();
        // Award?
        if (deservesAward()) {
            // Yes
            createAward();
        }
        // No

        // End
    }
}
